### PR TITLE
5 get/api/users

### PR DESCRIPTION
**Pull request by aDiallo21**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -76,4 +76,32 @@ describe("Nc News, testing API's", () => {
...
+                avatar_url: expect.any(String),
+              })
+            );
+          });
+        });
+    });
+    it("should respond with 404 - Not found when path is incorrect", () => {
+      request(app)
+        .get("/api/isers")
+        .expect(404)
```
Comment (Rayhaan2k9) [inline __tests__/app.test.js]: This should be a universal error test for any incorrect endpoint, not just for /api/users. You can put it in its own describe block

**Review by Rayhaan2k9 — APPROVED** [review]
Just a small suggestion but great work :)

File: __tests__/app.test.js (hunk 1 above)
Comment (aDiallo21) [reply __tests__/app.test.js]: Hi Rayhaan, thanks for the feedback! I've changed it now, is it good to merge?

**Review by leekli — APPROVED** [review]
Good to merge.

**Pull request merged**